ui/js/models/repository.js

```diff
@@ -298,7 +298,7 @@ treeherder.factory('ThRepositoryModel', [
 
             // filter out non-watched and unsupported repos to prevent repeatedly
             // hitting an endpoint we know will never work.
-            repoNames = _.filter(repoNames, function (repo) {
+            repoNames = repoNames.filter((repo) => {
```

Comment:
I just rebased my Neutrino v7 branch (whilst happens to also enabled a few more eslint checks) on master after this landed, and eslint reported:

```
ui\js\models\repository.js
  301:49  error  Expected to return a value at the end of arrow function  array-callback-return
```

I'm not sure whether lodash would have handled the missing return differently or not (and so whether this will cause a change in behaviour), but it would be good to fix this warning either way. I don't suppose you'd be up for opening a quick PR to do this? If you don't have time that's no problem, I can take a look :-)

Reply:
Oh, I can get this, that's a simple fix. Good catch on the new eslint rules, I was confused as to why you were getting linter errors and I wasn't. I'll get that in right now.

Reply:
As part of the Neutrino upgrade, I'm switching our eslint config to use airbnb-base with some things turned off (just until they are made to pass), in contrast with the current setup which is "most things off by default, apart from the few rules we've gotten around to turning on" hehe. Later (in bug 1183749) the list of airbnb-base rules turned off can be reduced.